cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala

```diff
@@ -198,3 +207,128 @@ class MultiAnomalyEstimatorSuite extends EstimatorFuzzing[MultivariateAnomalyEst
   override def modelReader: MLReadable[_] = DetectMultivariateAnomaly
 }
 
+
+class SimpleMultiAnomalyEstimatorSuite extends EstimatorFuzzing[SimpleMultiAnomalyEstimator] with MADUtils {
+
+  import MADUtils._
+  import spark.implicits._
+
+  lazy val fileLocation: String = DatasetUtils.madTestFile("mad_example.csv").toString
+  lazy val dataset: DataFrame = spark.read.format("csv").option("header", "true")
+    .load(fileLocation)
+  lazy val emptyDf: DataFrame = Seq("").toDF()
+
+  def simpleMultiAnomalyEstimator: SimpleMultiAnomalyEstimator = new SimpleMultiAnomalyEstimator()
+    .setSubscriptionKey(anomalyKey)
+    .setLocation("westus2")
+    .setOutputCol("result")
+    .setContainerName("madtest")
+    .setSourceDF(dataset)
+    .setStartTime(startTime)
+    .setEndTime(endTime)
+    .setConcurrency(5)
+
+  test("SimpleMultiAnomalyEstimator basic usage with connectionString") {
+
+    val smae = simpleMultiAnomalyEstimator.setSlidingWindow(200).setConnectionString(connectionString)
+    val model = smae.fit(emptyDf)
+    val diagnosticsInfo = smae.getDiagnosticsInfo
+    assert(diagnosticsInfo.variableStates.get.length.equals(3))
+
+    val result = model
+      .setSource(smae.getSource).setStartTime(startTime).setEndTime(endTime)
+      .setOutputCol("result")
+      .transform(emptyDf)
+      .withColumn("status", col("result.summary.status"))
+      .withColumn("variableStates", col("result.summary.variableStates"))
+      .select("status", "variableStates")
+      .collect()
+
+    assert(result.head.getString(0).equals("READY"))
+    assert(result.head.getSeq(1).length.equals(3))
+
+    madDelete(model.getModelId)
```

Comment:
Right now cleanup won't run unless tests pass (which can cause bloat of of objects). Consider using a try finally to ensure cleanup runs. Also, it might be good to put the delete model func on the model itself for easy cleanup.